### DISCUSSION TITLE
Relational Interfaces vs normal releases

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#12577] [Archive]
I have one question which I am wondering about, I am very curious why there is a division into normal releases and Relational Interfaces? There are 24 fixes that haven't been released in months and users keep reporting the same bugs, because they don't pay attention to such old reports. What if any of these fixes are buggy - and you have to work on the changes again.
When Relational Interfaces Rework # 12082 and Add Data Flows to Directus # 12522 will be released, there is no better solution on the market. Directus rules.
I know you guys are a small group of programmers, but I think so far not many have done so much for the data world!!!

**Comment by benhaynes** [comment]
I'm not sure I completely understand the question, but we have our normal releases every two weeks (typically on Thursdays), and we're also trying to get the Relational Fixes PR out as quickly as possible. No ETA on that yet though, as we have a lot going on at the moment. 🙂 

Let me know if I'm misunderstanding the question!

**Reply by abdonrd** [reply]
Maybe it's related with the milestones:

https://github.com/directus/directus/milestones

**Reply by koksikus** [reply]
Yes, that's what I meant about this division in milestones. I was just wondering why ready-made solutions cannot be transferred to normal releases. And that's it. The fact that there are over 25 patches ready for relational interfaces and there is still no release date. Your joint both answer is for me complete. Thanks and waiting for the relational releases 😉. Best regards.

**Comment by rijkvanzanten** [comment]
Ohh it's just for us to have a way to keep track of all the issues that are related to relational interfaces, nothing more! It's unrelated to the actual releases. It was mostly a way for us to keep track of what issues we had to solve in the Relational Interfaces Rework PR you already mentioned 🙂
